### PR TITLE
New Port: Icy Tower

### PR DESCRIPTION
**Pull request by Cebion**
Game Information
Title: Icy Tower
URL: https://github.com/Cebion/icytower

Controls:
| Button | Action |
|--|--| 
|DPAD| Move|
|A/B| Jump|

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] MuOS
[] Knulli (Optional)
[] Crossmix (Optional)

Architectures: 
[x] Aarch64
[X] Panfrost (Mainline)

Resolutions: 
[] 480x320 (Optional)
[X] 640x480
[X] 720x720 (RGB30) (Optional)
[X] Higher resolutions (e.g., 1280x720) 

Ready to Run!

### CODE REVIEW

File: ports/icytower/screenshot.png
Comment (tabreturn) [inline ports/icytower/screenshot.png:1]: Another pedantic one --
Is there some weird graphical glitch on the platform the dude is standing on?

**Review by tabreturn — APPROVED** [review]
Left some pedantic comments. Otherwise, looks good :)

File: ports/icytower/screenshot.png
Comment (Cebion) [reply ports/icytower/screenshot.png:1]: That's how it looks on my handheld. Took the screenshot directly on device :P

File: ports/icytower/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Notes
+Thanks to the [Roy E & RaMMicHaeL](https://github.com/royeldar/icytower) for remaking this game in allegro 5 and making it available for free!
```
Comment (Cebion) [inline ports/icytower/README.md:2]:
```suggestion
Thanks to the [Roy E & RaMMicHaeL](https://github.com/royeldar/icytower) for remaking this game in Allegro 5 and making it available for free!
```

File: ports/icytower/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+|--|--| 
+|DPAD| Move|
+|A/B| Jump|
+|X| Pause|
+
+## Compile
+
+```shell
+
+// Requirements compile allegro5 with SDL2 build
```
Comment (Cebion) [inline ports/icytower/README.md:16]:
```suggestion
// Requirements compile Allegro 5 with SDL2 build
```

**Review by tabreturn — APPROVED** [review]

File: ports/icytower/gameinfo.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./Icy Tower.sh</path>
+    <name>Icy Tower</name>
+    <desc>Reach new heights by jumping from platform to platform in the Icy Tower game!</desc>
+    <releasedate>20011222T000000</releasedate>
+    <developer>Anders Svensson</developer>
+    <publisher>Self-Published</publisher>
+    <genre>Platformer</genre>
+    <image>./screenshot.png</image>
```
Comment (kloptops) [inline ports/icytower/gameinfo.xml:11]: Wrong path?

```suggestion
    <image>./icytower/screenshot.png</image>
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]
BIngo bango

**Pull request merged**